pkg/compare/compare.go

```diff
@@ -792,11 +794,33 @@ func (obj InfoObject) Merged() (runtime.Object, error) {
 		}
 		obj.injectedObjFromTemplate = patched
 	}
+	obj.runInlineDiffFuncs()
 
 	omitFields(obj.injectedObjFromTemplate.Object, obj.FieldsToOmit)
 	return obj.injectedObjFromTemplate, err
 }
 
+func (obj InfoObject) runInlineDiffFuncs() {
+	for pathToKey, inlineDiffFunc := range obj.templateFieldConf {
+		listedPath, err := pathToList(pathToKey)
+		if err != nil {
+			return
```

Comment:
Or if we do `return`, that would mean it's a syntax error in the metadata or the template, right?  Perhaps this function should return `error` so we can output a relevant message to the user?

Reply:
I think continue and compiling a list errors would be a good idea.

Reply:
done, though some of these errors will never occur (or at least should) because they are detected when parsing the templates in the validation function